### PR TITLE
Fix IRC channel name

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: src/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+23 error If you do, this is most likely a problem with the bitcoin-unlimited-web package,
+23 error not with npm itself.
+23 error Tell the author that this fails on your system:
+23 error     node index.js
+23 error You can get information on how to open an issue for this project with:
+23 error     npm bugs bitcoin-unlimited-web
+23 error Or if that isn't available, you can get their info via:
+23 error     npm owner ls bitcoin-unlimited-web
+23 error There is likely additional logging output above.
+24 verbose exit [ 1, true ]
```
Comment (thofmann) [inline src/npm-debug.log]: Please remove this debug file

File: src/public/views/pages/why.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                                </div>
+                            </div>
+                        </div>
+                </div>
+                <Footer/>
+            </div>
+        );
+    }
+
+});
```
Comment (thofmann) [inline src/public/views/pages/why.jsx]: This is unrelated to the PR

Comment (sickpig) [inline src/public/views/pages/why.jsx]: sorry for that both why and the debug.log of course was not related to this PR

**Comment by sickpig** [comment]
@thofmann removed all the unneeded files
